server.ts

```diff
@@ -0,0 +1,45 @@
+const [, major] = process.version.match(/v(\d+)\.(\d+)\.(\d+)/);
+if (+major < 5) {
+  console.log("\x1b[31mWARNING: YOU SHOULD BE RUNNING THIS WITH NODE >= 5!! FOUND VERSION:", process.version, "\x1b[0m");
+  process.exit(1);
+}
+
+
+var bee    = require('beeline');
```

Comment:
its from one of my favorite authors (https://github.com/xavi-/beeline) :-) I choose it because its light weight and simpler to get going than express, but we could switch if there is any advantage in being consistent for this middleware.